OrbitGl/App.cpp

```diff
@@ -595,6 +595,7 @@ void OrbitApp::StopCapture() {
 }
 
 void OrbitApp::ClearCapture() {
+  set_selected_thread_id(0);
```

Comment:
0 is kernel process iirc. use -1 for no-porcess/thread-selected.

Reply:
done.